arch/arm/src/armv7-m/arm_vectors.c

```diff
@@ -51,6 +51,47 @@
 #  error ARMV7M_PERIPHERAL_INTERRUPTS must be defined to the number of I/O interrupts to be supported
 #endif
 
+#define ARMV7M_VECTAB_SIZE (ARMV7M_PERIPHERAL_INTERRUPTS + 16)
```

Comment:
why dup the code at nuttx/arch/arm/src/armv7-m/arm_ramvec_initialize.c

Reply:
Sorry, this commit(arm_vectors.c) is my misstaken, I will remove from this PR.
About arm_vector.c there is another PR : https://github.com/apache/nuttx/pull/12442, please refer it.

Reply:
@xiaoxiang781216 
I removed arm_vectors.c from this PR.